reaper-adm-extension/src/reaper_adm/exportaction_admsource-earvst.cpp

```diff
@@ -356,6 +364,26 @@ std::optional<double> EarVstExportSources::getValueFor(std::shared_ptr<admplug::
     return std::optional<double>();
 }
 
+bool EarVstExportSources::getEnvelopeBypassed(TrackEnvelope* env, ReaperAPI const& api)
+{
+    bool envBypassed = false;
+    char chunk[1024]; // A new envelope should only be about 100 bytes
```

Comment:
Is this assumption valid? https://forums.cockos.com/showthread.php?p=2142251 suggests envelope points are stored in the state chunk and something like the turning forest (S3A dataset available here https://cvssp.org/data/s3a/public/radiodrama_register.php) have a lot of object automation points (thousands?)